src/app/legacy/containers/PageHandlers/withRUM/index.jsx

```diff
@@ -2,6 +2,18 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import isLive from '#lib/utilities/isLive';
 
+// Note - if changing one of these constants, the other will also need to change
+// See https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
+const SCRIPT_SRC = 'https://client.rum.us-east-1.amazonaws.com/1.2.1/cwr.js';
+const SCRIPT_INTEGRITY =
+  'sha384-eCJxxyQILPZN5He1/UBN7E0nPIPBXIMDEPYubMaxQsCcQ+Vh8W9n6zRgQdqNgL0D';
```

Comment:
Should this be public or a secret?

Reply:
yep - this needs to be public - it gets sent to users' browsers as part of the script tag, eg:

`<script src="https://client.rum..." integrity="sha384-eCJ..." />`

The idea is that browsers download the script and regenerate this integrity checksum.  If they get a different result from us, that means the content of the script has changed since we generated the one in this PR.

That's a potential security issue, so the browser will not execute the script